mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/JobClusterActor.java

```diff
@@ -600,6 +601,7 @@ private Receive buildInitializedBehavior() {
                 // UNEXPECTED MESSAGES END //
                 .match(Terminated.class, this::onTerminated)
                 .matchAny(x -> {
+                    MDC.put("name", name);
```

Comment:
Doesn't this make assumptions about the thread context? Will these assumptions be held for Akka actors?

Reply:
Oh, I didn't consider it. So does it mean it is better to use ActorContext.getLog() to obtain the logger and clear the MDC after processing the current message?

Reply:
I don't think we want to use MDC for this.

Reply:
It seems I thought it the wrong way. Thank you for telling me and I will fix it after my busy final.